content/blog/2020/11/2020-11-03-gsoc-report.adoc

```diff
@@ -0,0 +1,163 @@
+This is the third year when the Jenkins organization sends extra GSoC swag,
+In the previous years the swag logistics was one of the most challenging tasks for org admins during the entire project.
+and we highly appreciate help from CDF with this part.
+As DevOps World presenters, the students have also received special edition speaker swag from CloudBees.
+
+image:/images/gsoc/2020/summary_report/swag.jpg[GSoC swag by the Jenkins project]
+
+Thanks a lot to link:https://opensource.google/[Google], link:https://cd.foundation/[Contrinuos Delivery Foundation (CDF)] and link:https://www.cloudbees.com/[CloudBees]!
+
+== Retrospective
+
+After completion of the coding phases,
+org admins have reached out to all GSoC 2020 participants to gather their feedback and suggestions,
+and we also recommended project teams to hold their own retrospective meetings.
+Such information is instrumental to continuously improving GSoC in the Jenkins community.
+We thank all contributors who shared their feedback!
+
+The organization-wide retrospective was organized as a survey and a series of retrospective meetings.
+You can find aggregated results in link:https://docs.google.com/document/d/1NIszUtuXmHiu8X2WrgAEQFK6aVodsmM4I0RSNRf4TS0/edit?usp=sharing[this Google Doc].
+Overall, we have got a very positive feedback from students and mentors.
+The GSoC framework in Jenkins has matured a lot during the previous years,
+and we have invested a lot in creating guidelines and recommendations for all parties.
+This year these guidelines helped a lot because all the expectations were known in advance.
+As usual, there is a lot to improve, especially with regards to the community bonding phases and cross-project communications.
+We are processing the feedback, and we will expand our documentation and the contributor onboarding plans next year.
+
+== Some personal notes
+
+I have been involved in leading and coordinating Google Summer of Code in open source projects since 2016.
+This year I visited the GSoC stand at FOSDEM and met a few organizers and former students.
+A few days after, I proposed participating in GSoC at the Jenkins contributor summit in Brussels,
+and several contributors supported this idea.
+we spent several hours to create the first Jenkins GSoC page and brainstorm on project ideas.
+We submitted our application, and we got accepted.
+Thanks a lot to the Google team that gave us a chance!
+
+It is great to work with the students and see how they explore the open source community and grow professionally as engineers.
+It is also awesome to see how some of them stay in the project and keep contributing,
+including becoming plugin maintainers and GSoC mentors
+But, for me, Google Summer of Code is not just about mentoring.
+It also helps a lot with the community bonding... for the existing community like Jenkins which has a lot of isolated sub-communities in plugins.
+Many maintainers work alone, and it is not the best place.
```

Comment:
Consider expanding "not the best place" with more specific information about the loneliness of open source development for many developers.
```suggestion
Many maintainers work alone, and it can be quite lonely working to maintain a plugin without feedback, developer ideas, and user interactions.
```